tests/clients/test_mysqlclient.py

```diff
@@ -217,3 +217,90 @@ def test_error_capture(self):
         self.assertEqual(db_span.data["mysql"]["stmt"], 'SELECT * from blah')
         self.assertEqual(db_span.data["mysql"]["host"], testenv['mysql_host'])
         self.assertEqual(db_span.data["mysql"]["port"], testenv['mysql_port'])
+
+    def test_connect_cursor_ctx_mgr(self):
+        result = None
+        with tracer.start_active_span("test"):
+            with self.db as connection:
+                with connection.cursor() as cursor:
+                    result = cursor.execute("""SELECT * from users""")
+                    cursor.fetchone()
+
+        assert result >= 0
```

Comment:
This assert is for production codes runtime anomalies. For unit tests please use from the `unittest` module, if nothing else fits then [assertTrue](https://docs.python.org/3/library/unittest.html?highlight=asserttrue#unittest.TestCase.assertTrue)